src/utils/atomWithReducer.ts

```diff
@@ -3,17 +3,17 @@ import type { WritableAtom } from 'jotai'
 
 export function atomWithReducer<Value, Action>(
   initialValue: Value,
-  reducer: (v: Value, a?: Action) => Value
+  reducer: (state: Value, action?: Action) => Value
```

Comment:
```suggestion
  reducer: (previousValue: Value, action?: Action) => Value
```
prefer "value" to "state".